Alias "deptrac" is outdated - repository moved to `deptrac/deptrac`

The `deptrac` repository has moved from `qossmic/deptrac` to `deptrac/deptrac`.

When I try to install `deptrac` with `phive` using the alias in `data/repositories.xml`:

```bash
phive install deptrac
```

I get the following error:

```
Phive 0.15.3 - Copyright (C) 2015-2025 by Arne Blankerts, Sebastian Heuer and Contributors
Fetching repository list
Downloading https://phar.io/data/repositories.xml
Downloading https://api.github.com/repos/qossmic/deptrac/releases?per_page=100
[ERROR]    No matching release found for deptrac!
```

--- a/data/repositories.xml
+++ b/data/repositories.xml
@@ -45,8 +45,8 @@
     <phar alias="dephpend" composer="dephpend/dephpend">
         <repository type="github" url="https://api.github.com/repos/mihaeu/dephpend/releases"/>
     </phar>
-    <phar alias="deptrac" composer="qossmic/deptrac">
-        <repository type="github" url="https://api.github.com/repos/qossmic/deptrac/releases"/>
+    <phar alias="deptrac" composer="deptrac/deptrac">
+        <repository type="github" url="https://api.github.com/repos/deptrac/deptrac/releases"/>
     </phar>
     <phar alias="doctum" composer="code-lts/doctum">
         <repository type="github" url="https://api.github.com/repos/code-lts/doctum/releases"/>

